admin_guide/topics/proc_adding-hosts.adoc

```diff
@@ -0,0 +1,156 @@
+
+[IMPORTANT]
+====
+The *_scaleup.yml_ playbook configures only the new host. It does not update 
+*_NO_PROXY_* in master services, and it does not restart master services.
+====
+
+You must have an existing inventory file,for example *_/etc/ansible/hosts_*,
+that is representative of your current cluster configuration in order to run the
+*_scaleup.yml_* playbook.
+ifdef::openshift-enterprise[]
+If you previously used the `atomic-openshift-installer` command to run your
+installation, you can check *_~/.config/openshift/hosts_* for the last inventory
+file that the installer generated and use that file as your inventory file. You
+can modify this file as required. You must then specify the file location with 
+`-i` when you run the `ansible-playbook`.
+endif::[]
+
+[IMPORTANT]
+====
+See the
+xref:../scaling_performance/cluster_limits.adoc#scaling-performance-cluster-limits[cluster
+limits] section for the recommended maximum number of nodes.
+====
+
+[discrete]
+== Procedure
+
+. Ensure you have the latest playbooks by updating the *atomic-openshift-utils*
+package:
++
+----
+# yum update atomic-openshift-utils
+----
+
+. Edit your *_/etc/ansible/hosts_* file and add *new_<host_type>* to the
+*[OSEv3:children]* section:
++
+For example, to add a new node host, add *new_nodes*:
++
+----
+[OSEv3:children]
+masters
+nodes
+new_nodes
+----
++
+To add new master hosts, add *new_masters*.
+
+. Create a *[new_<host_type>]* section to specify host information for the new 
+hosts. Format this section like an existing section, as shown in the following
+example of adding a new node:
++
+----
+[nodes]
+master[1:3].example.com
+node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
+node2.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
+infra-node1.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+infra-node2.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+
+[new_nodes]
+node3.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
+----
++
+See
+xref:../install_config/install/advanced_install.adoc#advanced-host-variables[Configuring
+Host Variables] for more options.
++
+When adding new masters, add hosts to both the *[new_masters]* section and the 
+*[new_nodes]* section to ensure that the new master host is part of
+the OpenShift SDN.
++
+----
+[masters]
+master[1:2].example.com
+
+[new_masters]
+master3.example.com
+
+[nodes]
+master[1:2].example.com
+node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
+node2.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
+infra-node1.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+infra-node2.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+
+[new_nodes]
+master3.example.com
+----
++
+Masters are also automatically marked as unschedulable for pod placement by the
+installer.
```

Comment:
Hmm is this accurate still? I think this went away with 3.9...

EDIT: yup:

https://docs.openshift.com/container-platform/3.9/install_config/install/advanced_install.html#marking-masters-as-unschedulable-nodes

Reply:
Yup. Will do. And I'm going to make a note to check the rest of the instances of "unschedulable" in master.